src/components/MoviePoster/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+import NoPoster from '../MovieNoPoster';
+
+function showPoster(src, alt) {
+  if (!src) {
```

Comment:
Just a note. Some movie posters in `MovieApi` have invalid/broken links and in such scenario MoviePoster can render invalid result

Reply:
Yes, you are right. I am revising my login for this. Thanks.